apps/web-mzima-client/src/app/feed/feed.component.ts

```diff
@@ -213,7 +213,7 @@ export class FeedComponent extends MainViewComponent implements OnInit, OnDestro
         this.urlAfterInteractionWithFilters = '';
         //-----------------------------------
 
-        this.activeCard.scrollCountHandler({ task: 'increment' });
+        this.activeCard.scrollCountHandler({ task: 'reset' });
 
         this.masonryUpdateOnModeSwitch({ userEvent: this.userEvent });
```

Comment:
@Ifycode I may have messed up something else, but I could not see what. This increment was causing the post to not scroll into view after update 🤔

Reply:
Ok, I see the issue. 

The `feedview_postObj`'s post id in the local storage is the same as the post id in the browser URL (i.e. the post id is stored so that we can target the associated postcard for scrolling).

Postcard is only to scroll to view at the point when `scroll_count` is at number `1`.

The `reset` task removes `scroll_count` property from local storage. So applying reset on navigation end would prevent scroll since there's no scroll count to make reference to, though `feedview_postObj` has the stored post id prepared.

> Comment continued on line `888` below.

Reply:
Good catch @Ifycode !